src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -75,7 +75,11 @@ void buildSearchConceptsQueryEmpty(CloudPlatform platform) {
         SearchConceptsQueryBuilder.buildSearchConceptsQuery(
             domainOption, "", s -> s, CloudPlatformWrapper.of(platform));
     String expected =
-        "c.concept_name, c.concept_id FROM concept AS c " + "WHERE c.domain_id = 'Condition' ";
+        "c.concept_name, c.concept_id, COUNT(DISTINCT c0.person_id) AS count "
```

Comment:
Testing for the SQL contents works but I worry that we don't have integration tests for much of this code. Maybe it's something we can think about later on, once most of the query code is done.

Reply:
Hopefully, connecting to a local UI with integrated data would be a great way of testing this.

Reply:
That would work for validating a PR change, but I'd really want an automated test or at least a script that we can run manually once in a while

Reply:
I agree, I think adding back integration tests will be a great ticket.